### ISSUE TITLE
How to specify the port

### ISSUE THREAD

**Issue by Vanlightly** [CloudKarafka/KafkaHttpReporter#2]
The reporter always starts up listening on port 19092. I can't figure out how to set the port in configuration. Looking at the clojure code it seems like it might be possible but I do not understand clojure at all.

**Comment by Sugaroverdose** [comment]
It seems like that you need to specify port in kafka server config, option `kafka_http_reporter.port`

**Comment by Vanlightly** [comment]
Yes that works. The issue is that when using Docker it is very convenient to use environment variables to set configuration values. This config has underscores in it which get converted to dots. So kafka_http_reporter.port it turned into http.reporter.port.

Could we change the config from kafka_http_reporter.port to kafka.http.reporter.port? Then with Docker we can use the environment variable KAFKA_KAFKA_HTTP_REPORTER_PORT.

**Comment by Sugaroverdose** [comment]
You can easily try it out by changing single line
https://github.com/CloudKarafka/KafkaHttpReporter/blob/d1937224855fa0f7f2c3ea615d4158fc96f94ed7/src/cloudkarafka/kafka_http_reporter.clj#L91

But it's better just to remove underscores instead of changing them to dots

**Comment by Vanlightly** [comment]
Yes that works. I changed it to kafkahttpreporter.port and set the environment variable to KAFKA_KAFKAHTTPREPORTER_PORT: 19093

In the logs I get [INFO] KafkaHttpReporter: Starting HTTP server listening on port  19093.

The CloudKarafka MGMT assumes that the port is the advtertised listener port with a 1 at the front (so adv listener port 9093 means HTTP reporter port of 19093). So just need to ensure that the port is configured to that. That restriction seems fine, I can't think of a reason why that might cause people problems.

**Comment by Vanlightly** [comment]
I created a PR https://github.com/CloudKarafka/KafkaHttpReporter/pull/3
